generateData.ts

```diff
@@ -0,0 +1,52 @@
+class Row {
+    message: string;
+    inferredLabel: string;
+    correctlyClassified: boolean;
+    comment: string;
+
+    constructor(message: string, inferredLabel: string) {
+        this.message = message
+        this.inferredLabel = inferredLabel
+    }
+
+    addHumanCheck(correctlyClassified: boolean, comment?: string) {
+        this.correctlyClassified = correctlyClassified
+        this.comment = comment
+    }
+}
+
+function pickRandomLetter(): string {
+    let letterIndex = Math.floor(Math.random() * 26)
+
+    return "abcdefghijklmnopqrstuvwxyz".charAt(letterIndex)
```

Comment:
This implementation relies on an the implicit invariant that len("abcd...") == 26, it would be better practice to do something like (psuedo code)

chars = "abcdefghijklmnopqrstuvwxyz";
letterIndex = Math.floor(Math.random() * chars.length)